### PR TITLE
[13.0][MIG] mass_mailing_partner: Migration to 13.0

### PR DESCRIPTION
**Pull request by manuelcalerosolis**
This is the migration to v 13.0 module mass_mailing_partner

@Tecnativa TT19777

### CODE REVIEW

File: mass_mailing_partner/tests/test_mail_mass_mailing_contact.py [hunk 1]
```diff
@@ -104,7 +105,7 @@ def test_onchange_partner(self):
             "category_id": [(6, 0, (category_8 | category_11).ids)],
         }
         partner = self.create_partner(partner_vals)
-        with self.env.do_in_onchange():
-            contact.partner_id = partner
-            contact._onchange_partner_mass_mailing_partner()
-            self.check_mailing_contact_partner(contact)
+        # with self.env.do_in_onchange():
```
Comment (pedrobaeza) [inline mass_mailing_partner/tests/test_mail_mass_mailing_contact.py]: You can remove this line directly.

File: mass_mailing_partner/tests/test_partner_mail_list_wizard.py [hunk 2]
```diff
@@ -24,6 +25,7 @@ def test_add_to_mail_list(self):
         # This line does not create a new contact
         wizard.add_to_mail_list()
         self.assertEqual(len(self.partner.mass_mailing_contact_ids), 1)
+
```
Comment (pedrobaeza) [inline mass_mailing_partner/tests/test_partner_mail_list_wizard.py]: Why this extra empty line? PEP8 discourages to introduce these lines inside a method.

**Review by pedrobaeza — APPROVED** [review]

File: mass_mailing_partner/tests/test_mail_mass_mailing_contact.py (hunk 1 above)
Comment (manuelcalerosolis) [reply mass_mailing_partner/tests/test_mail_mass_mailing_contact.py]: ok

**Review by ernestotejeda — COMMENTED** [review]
Thanks @manuelcalerosolis 

Could you check that:
Peek 2020-02-10 15-58

File: mass_mailing_partner/tests/test_partner_mail_list_wizard.py (hunk 2 above)
Comment (manuelcalerosolis) [reply mass_mailing_partner/tests/test_partner_mail_list_wizard.py]: Deleted!

**Comment by pedrobaeza** [comment]
@manuelcalerosolis instead of adding new commit, better to simply amend previous commit. Check also that the problem depicted by @ernestotejeda is not solved and my other comment is not attended as well.

**Review by chienandalu — APPROVED** [review]
:+1: Thanks :) Tested on runbot and code review

**Comment by pedrobaeza** [comment]
@manuelcalerosolis please squash your migration commits into one and I'll merge.

**Review by ernestotejeda — APPROVED** [review]

**Pull request merged**